docs/conf.py

```diff
@@ -3,56 +3,56 @@
 import sys
 from os import path
 
-sys.path.insert(0, path.abspath('..'))
-
-project = 'DocArray'
-slug = re.sub(r'\W+', '-', project.lower())
-author = 'Jina AI'
-copyright = 'Jina AI Limited. All rights reserved.'
-source_suffix = ['.rst', '.md']
-master_doc = 'index'
-language = 'en'
-repo_dir = '../'
+sys.path.insert(0, path.abspath(".."))
```

Comment:
we usually prefer single quote

Reply:
Oh these changes are from black. I can revert them but the next person who tries to black this section of code will have the same changes

Reply:
Hrmm but it will conflict with ur PR on redirects. Ill revert them here for now. Can blacken later if u wish